CITATION.cff

```diff
@@ -0,0 +1,65 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: perprof-py
+message: Please cite this software using the metadata from 'preferred-citation'.
+type: software
+authors:
+  - given-names: Abel
+    family-names: Soares Siqueira
+    email: abel.s.siqueira@gmail.com
+    affiliation: Netherlands eScience Center
+    orcid: 'https://orcid.org/0000-0003-4451-281X'
+  - given-names: Raniere
+    family-names: Gaia Costa da Silva
+    email: raniere.silva@my.cityu.edu.hk
+    affiliation: City University of Hong Kong
+    orcid: 'https://orcid.org/0000-0002-8381-3749'
+  - given-names: Luiz-Rafael
+    family-names: Santos
+    email: lrsantos11@gmail.com
+    affiliation: Universidade Federal de Santa Catarina
+    orcid: 'https://orcid.org/0000-0002-0083-5871'
+identifiers:
+  - type: doi
+    value: 10.5281/zenodo.593666
+    description: Zenodo concept DOI
+repository-code: 'https://github.com/ufpr-opt/perprof-py'
```

Comment:
Maybe you want to move it to another organisation.

Reply:
Can you suggest one?

Reply:
We moved it from https://github.com/lpoo to https://github.com/ufpr-opt because none of us were at UNICAMP and Abel was at UFPR. Now, Abel is not at UFPR and I don't know he signed a collaboration agreement or not.

We could keep the project under Abel's account or create a independent organisation or join one existing organisation (e.g. [pyOpenSci](https://www.pyopensci.org/python-packages/))